backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -793,7 +802,9 @@ public DiagnosticReport convertToDiagnosticReport(
     var diagnosticReport =
         new DiagnosticReport()
             .setStatus(status)
-            .setEffective(new DateTimeType(dateTested))
+            .setEffective(
+                new DateTimeType(
+                    dateTested, TemporalPrecisionEnum.MILLI, TimeZone.getTimeZone("utc")))
```

Comment:
see above comment - this just forces the messages to look the same when run on systems with different default precision/time zone